mbed_devices/mbed_tools/cli.py

```diff
@@ -52,9 +54,10 @@ def _build_json_output(devices: Iterable[Device]) -> str:
             mbed_target_data = {
                 "product_code": mbed_target.product_code,
                 "board_type": mbed_target.board_type,
-                "platform_name": mbed_target.platform_name,
+                "board_name": mbed_target.board_name,
                 "mbed_os_support": mbed_target.mbed_os_support,
                 "mbed_enabled": mbed_target.mbed_enabled,
+                "build_targets": _get_build_targets_str(mbed_target, ","),
```

Comment:
In JSON land this would be a list not a comma separated string.

Reply:
Will do.